package.json

```diff
@@ -4,12 +4,15 @@
   "description": "React / Webpack / Babel framework for quick set up",
   "main": "index.js",
   "scripts": {
-    "build": "webpack --config ./webpack.prod.config.js --progress --colors",
+    "clean": "rm -rf ./dist/*",
+    "build": "webpack --config ./webpack.config.js --progress --colors",
+    "build:prod": "NODE_ENV='production' webpack --config ./webpack.config.js --progress --colors",
     "start": "node ./app.js",
     "dev": "webpack-dev-server --inline --hot",
```

Comment:
do we need this? if not, we could change `styleguide` to use `dev` in keeping with the other repos?